--- libofx-0.10.9/configure.ac.orig	2022-10-03 00:00:00.000000000 +0200
+++ libofx-0.10.9/configure.ac	2022-10-03 00:00:01.000000000 +0200
@@ -45,7 +45,7 @@
 			  [Define if GCC visibility extensions are supported])
 	VISIBILITY_FLAGS="-fvisibility=hidden"
 	CFLAGS="$VISIBILITY_FLAGS $CFLAGS"
-	CXXFLAGS="$VISIBILITY_FLAGS -fvisibility-inlines-hidden $CFLAGS"
+	CXXFLAGS="$VISIBILITY_FLAGS -fvisibility-inlines-hidden $CXXFLAGS"
 fi

 ## Pass -DIN_LIBOFX to the compiler so we can detect it and include config.h
