--- libofx-0.9.1/ofxdump/ofxdump.cpp.wiget	2009-08-08 15:30:42.000000000 +0200
+++ libofx-0.9.1/ofxdump/ofxdump.cpp	2009-08-08 15:31:26.000000000 +0200
@@ -27,6 +27,8 @@
  *   (at your option) any later version.                                   *
  *                                                                         *
  ***************************************************************************/
+#include <cstring>
+#include <cstdlib>
 #include <iostream>
 #include <iomanip>
 #include <string>
--- libofx-0.9.1/lib/ofx_utilities.cpp.wiget	2009-08-08 15:22:31.000000000 +0200
+++ libofx-0.9.1/lib/ofx_utilities.cpp	2009-08-08 15:25:26.000000000 +0200
@@ -26,6 +26,8 @@
 #include <locale.h>
 #include "messages.hh"
 #include "ofx_utilities.hh"
+#include <cstdlib>
+#include <ctime>
 
 #ifdef OS_WIN32
 # define DIRSEP "\\"
--- libofx-0.9.1/lib/ofx_request_accountinfo.cpp.wiget	2009-08-08 15:29:22.000000000 +0200
+++ libofx-0.9.1/lib/ofx_request_accountinfo.cpp	2009-08-08 15:29:40.000000000 +0200
@@ -21,6 +21,7 @@
 #include <config.h>
 #endif
 
+#include <cstdlib>
 #include <string>
 #include "libofx.h"
 #include "ofx_request_accountinfo.hh"
--- libofx-0.9.1/lib/ofx_request_statement.cpp.wiget	2009-08-08 15:29:55.000000000 +0200
+++ libofx-0.9.1/lib/ofx_request_statement.cpp	2009-08-08 15:30:08.000000000 +0200
@@ -21,6 +21,7 @@
 #include <config.h>
 #endif
 
+#include <cstdlib>
 #include <string>
 #include "libofx.h"
 #include "ofx_utilities.hh"
--- libofx-0.9.1/lib/ofx_request.cpp.wiget	2009-08-08 15:25:55.000000000 +0200
+++ libofx-0.9.1/lib/ofx_request.cpp	2009-08-08 15:27:44.000000000 +0200
@@ -21,6 +21,7 @@
 #include <config.h>
 #endif
 
+#include <cstring>
 #include <string>
 #include "messages.hh"
 #include "libofx.h"
