--- libofx-0.8.0/lib/Makefile.am.orig	2005-04-21 15:48:56.000000000 +0200
+++ libofx-0.8.0/lib/Makefile.am	2006-01-24 20:44:17.612598000 +0100
@@ -42,7 +41,7 @@
 	-I${OPENSPINCLUDES} \
 	-DMAKEFILE_DTD_PATH=\"${LIBOFX_DTD_DIR}/\"
 
-libofx_la_LIBADD = @LIBOBJ@ ${OPENSPLIBS} -lstdc++
+libofx_la_LIBADD = @LIBOBJS@ ${OPENSPLIBS} -lstdc++
 libofx_la_LDFLAGS = -version-info @LIBOFX_SO_CURRENT@:@LIBOFX_SO_REVISION@:@LIBOFX_SO_AGE@
 
 
--- libofx-0.8.0/lib/Makefile.in.orig	2005-07-31 20:21:10.000000000 +0200
+++ libofx-0.8.0/lib/Makefile.in	2006-01-24 20:44:20.036749500 +0100
@@ -250,7 +255,7 @@
 	-I${OPENSPINCLUDES} \
 	-DMAKEFILE_DTD_PATH=\"${LIBOFX_DTD_DIR}/\"
 
-libofx_la_LIBADD = @LIBOBJ@ ${OPENSPLIBS} -lstdc++
+libofx_la_LIBADD = @LIBOBJS@ ${OPENSPLIBS} -lstdc++
 libofx_la_LDFLAGS = -version-info @LIBOFX_SO_CURRENT@:@LIBOFX_SO_REVISION@:@LIBOFX_SO_AGE@
 all: all-am
 
